ibis/backends/tests/test_vectorized_udf.py

```diff
@@ -352,7 +345,7 @@ def test_elementwise_udf_destruct_exact_once(backend, alltypes):
             output_type=dt.Struct(['col1', 'col2'], [dt.double, dt.double]),
         )
         def add_one_struct_exact_once(v):
-            key = v.iloc[0]
+            key = v.loc[0]
```

Comment:
Why this change?

Reply:
Dask has no support for `iloc`.

Reply:
Hmm. See my other comments. The object passed to these UDFs should be pandas object not dask objects..

Reply:
Hmm. That's a good point - I need to think about this. Right now we're applying the function and trusting dask to handle it. 

I'm not sure how you apply a function across independent Series? There's no guarantee that they're partitioned the same or on the same workers.